docs/jdbc.md

```diff
@@ -0,0 +1,495 @@
+# Connecting to SQL Server using JDBC
+
+## Setup and Usage
+
+### Adding the library as a dependency
+
+<!-- {x-release-please-start-version} -->
+
+### Maven
+Include the following in the project's `pom.xml`:
+
+##### Mysql
+```maven-pom
+<dependency>
+    <groupId>com.google.cloud.sql</groupId>
+    <artifactId>mysql-socket-factory-connector-j-8</artifactId>
+    <version>1.13.1</version>
+</dependency>
+```
+##### Maria DB
+
+```maven-pom
+<dependency>
+    <groupId>com.google.cloud.sql</groupId>
+    <artifactId>mariadb-socket-factory</artifactId>
+    <version>1.13.1</version>
+</dependency>
+```
+##### Postgres
+
+Include the following in the project's `pom.xml`:
+
+```maven-pom
+<dependency>
+    <groupId>com.google.cloud.sql</groupId>
+    <artifactId>postgres-socket-factory</artifactId>
+    <version>1.13.1</version>
+</dependency>
+```
+
+#### Gradle
+Include the following the project's `build.gradle`
+
+##### Mysql
+```gradle
+compile 'com.google.cloud.sql:mysql-socket-factory-connector-j-8:1.13.1'
```

Comment:
Let's use <LATEST_VERSION> or whatever the release-please syntax requires on all of these, so we're always pointing at the latest version.

Reply:
@ttosta-google just landed a change so release-please will automatically update this doc to the most recent released version, so we need to use <LATEST_VERSION> anymore.